rs/pocket_ic_server/src/state_api/state.rs

```diff
@@ -764,6 +764,30 @@ impl ApiState {
             (resp.status(), resp)
         }
 
+        async fn handler_api_subnet(
+            api_version: ApiVersion,
+            replica_url: String,
+            subnet_id: SubnetId,
+            endpoint: &str,
+            bytes: Bytes,
+        ) -> (StatusCode, Response<Incoming>) {
+            let client =
+                Client::builder(hyper_util::rt::TokioExecutor::new()).build(HttpConnector::new());
```

Comment:
Are the resource needs of this TokioExecutor ok?

Reply:
We use the same pattern for the existing handlers, too, e.g., handler_api_canister and handler_status (above), so this shouldn't create issues we don't see so far.